Google.Api.Generator/Generation/MethodDetails.cs

```diff
@@ -220,6 +220,28 @@ public ServerStreaming(ServiceDetails svc, MethodDescriptor desc) : base(svc, de
             public Typ AsyncEnumeratorTyp { get; }
         }
 
+        public sealed class ClientStreaming : MethodDetails, IStreaming
+        {
+            public ClientStreaming(ServiceDetails svc, MethodDescriptor desc) : base(svc, desc)
+            {
+                ApiCallTyp = Typ.Generic(typeof(ApiClientStreamingCall<,>), RequestTyp, ResponseTyp);
+                AbstractStreamTyp = Typ.Nested(svc.ClientAbstractTyp, $"{SyncMethodName}Stream");
+                ImplStreamTyp = Typ.Nested(svc.ClientImplTyp, $"{SyncMethodName}StreamImpl");
+                StreamingSettingsName = $"{desc.Name}StreamingSettings";
+                ModifyStreamingCallSettingsMethodName = $"Modify_{RequestTyp.Name}CallSettings";
+                ModifyStreamingRequestMethodName = $"Modify_{RequestTyp.Name}Request";
+                AsyncEnumeratorTyp = Typ.Generic(typeof(AsyncResponseStream<>), ResponseTyp);
```

Comment:
If this is client-streaming, why is there a response stream? Wouldn't we just get one response at the end?

Reply:
👍